### ISSUE TITLE
Chargements des articles

### ISSUE THREAD

**Issue by cobalt74** [barbogogo/leedReader#44]
Hello. J'utilise a fond l'appli depuis quelques jours avec la tablette et c'est top. Bravo.
2 suggestions 
- un compteur des non lus a coté du titre LeedReader
- une sorte de lasyloading quand on arrive a la fin des x articles chargés

**Comment by cobalt74** [comment]
- Partage d'un article avec shaarli
- pouvoir augmenter la taille de la police des articles
- les dossiers avec un titre long passe sur une seconde ligne mais la police est coupée sur la deuxième ligne 

Je rajoute ;)

**Comment by cobalt74** [comment]
- Pouvoir mettre un article en favoris lors de sa lecture
- diminuer un peu la taille de la police des dossiers, c'est trop gros :)

**Comment by cobalt74** [comment]
- le message pour mettre tous les messages comme non lus est en anglais.

**Comment by barbogogo** [comment]
Salut @cobalt74,
Merci pour ton retour en ce qui concerne l'utilisation sur tablette. Tu pourrait m'envoyer une capture d'écran (je n'ai pas fait de tests sur une tablette et je voudrais voir à quoi ça peut ressembler :smile:) ?

Merci pour toutes ces idées, j'ai ajouté mes commentaires en gras !
Je note : 
- [ ] un compteur des non lus a coté du titre LeedReader
- [x] une sorte de lasyloading quand on arrive a la fin des x articles chargés **Oui, j'y ai pensé, mais je n'ai pas encore commencé**
- [ ] pouvoir augmenter la taille de la police des articles **Proposer différents thèmes dans les paramètres**
- [ ] les dossiers avec un titre long passe sur une seconde ligne mais la police est coupée sur la deuxième ligne
- [x] Pouvoir mettre un article en favoris lors de sa lecture **Oui, il manque un menu pour mettre un article en favoris et en non lu lors de sa lecture**
- [x] diminuer un peu la taille de la police des dossiers, c'est trop gros :) **C'est peut être lié à la tablette, non ? Mais j'y penserai !!!**
- [x] le message pour mettre tous les messages comme non lus est en anglais. **Arf... mauvaise gestion du multi-langue...**

En ce qui concerne le partage avec Shaarli, c'est tout à fait possible avec l'application [Shaarli](https://play.google.com/store/apps/details?id=com.manatlan.tools.share). Il suffit ensuite de partager un article et de choisir l'application Shaarli dans la liste :smile:

**Comment by cobalt74** [comment]
Super pour shaarli j'avais pas vu la petite ampli Android qui me convient parfaitement.
Du coup je viens de voir un truc bizarre, on peut partager sur la liste des articles! :)
Faut que je regarde pour te faire quelques screenshots tablette. Je sais pas faire.

**Comment by barbogogo** [comment]
Salut @cobalt74, [LeedReader-v1.4.1](http://www.barbogogo.fr/?p=709) est disponible et prend en compte les points fermés ci-dessus.

**Comment by cobalt74** [comment]
J'ai plus de lasyloading sur la 1.4.3
Mon utilisation du chargement des articles ce fait plutôt l'osque je glisse avec le doigt dun article a l'autre. Ou alors j'ai halluciné car c'est pas encore fait !

**Comment by barbogogo** [comment]
Si, normalement c'est implémenté...
Je regarde ça car je n'ai rien touché de ce côté...

Tu as changé de version d'API ?

**Comment by cobalt74** [comment]
Bonne question !? J'ai pas changé l'API depuis la 1.4 je crois :)

**Comment by barbogogo** [comment]
Est-ce que tu peux essayer avec l'[API v0.10](https://github.com/barbogogo/leed-API/tree/152bbba827155a3ea40d803a03db92c6bb86cf1f) ?

**Comment by cobalt74** [comment]
Oué mais non!
J'ai un petit soucis aussi,'je lance l'appli et j'ai chargement en cours qui reste jusqu'à je clic sur l'écran qui me dirige vers les paramètres. Je sors du parametrage et hop les articles se chargent.

**Comment by barbogogo** [comment]
ok,
donc toujours pas de lasyloading... je regarde.
Oui, j'ai vu le même problème : sur la v1.4.3, il faut appuyer sur l'écran pour rentrer dans les paramètres puis quitter les paramètres pour que les informations se chargent... je regarde.

**Comment by barbogogo** [comment]
Ouverture du ticket https://github.com/barbogogo/leedReader/issues/48, sur le non chargement des données lors de l'ouverture de l'application.

**Comment by cobalt74** [comment]
c'est good sur le chargement.
pour le lasyloading, ça remarche mais j'ai une erreur API lorsque j'arrive à la fin je crois.

**API Connection Erreur 3**
No value for articles

**Comment by cobalt74** [comment]
en faite il serait sympa d'avoir le lasyloading sur la lecture de l'article suivant.
je m'explique : j'ai ma liste de 20 articles affichés, je clic sur le premier, je lis et passe au suivant avec un glissé du doigt mais arrivé au dernier, je suis obligé de cliquer sur rafraichir pour afficher la suite.

**Comment by barbogogo** [comment]
Salut,
Merci pour ce retour de bug.
Ok, en effet le lazy loading n'est pas implémenté lors de la lecture des articles. Très bonne idée ;-)

**Comment by barbogogo** [comment]
Salut @cobalt74,

Est-ce que tu rencontres toujours le problème : "les dossiers avec un titre long passe sur une seconde ligne mais la police est coupée sur la deuxième ligne" ?

**Comment by cobalt74** [comment]
bonne question ... @barbogogo je regarde ce soir mais je ne crois pas.
Par contre, j'ai toujours le problème de lazyloading sur l'article suivant. tu l'as implémenté ?

**Comment by barbogogo** [comment]
@cobalt74 
Non, je n'ai pas implémenté le lazy loading au niveau de la lecture des articles... seulement au niveau du défilement des flux...
Mais je vais m'y mettre...

**Comment by cobalt74** [comment]
cool ! :+1:  J'ai hate de tester.
